src/main/java/com/google/gcp_variant_transforms/library/SchemaGeneratorImpl.java

```diff
@@ -99,10 +99,11 @@ protected TableFieldSchema createRecordField(VCFHeader vcfHeader, String fieldNa
    * @return an INFO or FORMAT Field
    */
   @VisibleForTesting
-  protected TableFieldSchema convertCompoundHeaderLineToField(VCFCompoundHeaderLine headerLine) {
+  protected TableFieldSchema convertCompoundHeaderLineToField(VCFCompoundHeaderLine headerLine,
+                                                              boolean isInfo) {
```

Comment:
I think we need to check that, since this method is called multiple times during the schema generator, if the `countType = A` and is Info field, we set as `nullable`, if the field is `FORMAT` with `count=A`, we need to set as repeated.